### DISCUSSION TITLE
Problema com o sendgrid[RESOLVIDO]

### DISCUSSION THREAD

**Discussion by Marcuusvp** [balta-io/forum#62] [Tira Dúvidas] [answered]
Curso 2811 módulo da criação de serviço de email. Não consegui me cadastrar no sendGrid como o balta recomenda no video e fiz cadastro no sendinblue. Pra isso funcionar eu preciso ter um dominio? não entendi isso direito na aula, posso colocar meu proprio email pra na prop fromEmail?
Fiz tudo conforme a aula mas estou tendo um erro no email service de System.Net.Mail.SmtpException: The SMTP server requires a secure connection or the client was not authenticated. The server response was: Error: authentication Required

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @Marcuusvp como vai? Não precisa ter um domínio, só uma conta no SendGrid: https://sendgrid.com/free/
Feito isto, basta seguir como na aula e gerar a chave necessária.

Este erro que você teve não seria por conta da porta? Está utilizando a mesma do curso?
